src/commands/deploy.ts

```diff
@@ -165,8 +166,12 @@ async function verifyRuntimeIsCompatible(localRuntime: ProjectRuntime, ui: IAzur
                 telemetryProperties.cancelStep = 'learnMoreRuntime';
                 throw new UserCancelledError();
             } else {
-                outputChannel.appendLine(localize('azFunc.updateFunctionRuntime', 'Updating FUNCTIONS_EXTENSION_VERSION to "{0}"...', localRuntime));
-                appSettings.properties.FUNCTIONS_EXTENSION_VERSION = localRuntime;
+                const newAppSettings: { [key: string]: string } = await getCliFeedAppSettings(localRuntime);
+                for (const key of Object.keys(newAppSettings)) {
+                    const value: string = newAppSettings[key];
+                    outputChannel.appendLine(localize('updateFunctionRuntime', 'Updating "{0}" to "{1}"...', key, value));
```

Comment:
Will this message make sense without saying what it's updating?

Reply:
The 'key' should explain what it's updating